compose-configuration.js

```diff
@@ -0,0 +1,89 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { WebpackPluginServe } = require("webpack-plugin-serve");
+const { merge } = require("webpack-merge");
+const parts = require("./webpack.parts");
+
+const commonConfig = {
+  plugins: [new HtmlWebpackPlugin({ filename: "iframe.html" })],
+  resolve: {
+    extensions: [".ts", ".tsx", ".js", ".jsx", ".json"],
+    alias: {
+      react: path.resolve(__dirname, "node_modules/react"),
+    },
+    // Added this for Chromatic, perhaps we should generalise the config extension
+    fallback: { path: require.resolve("path-browserify") },
```

Comment:
Can we move this to the Chromatic config now that it merges configs recursively?

I'm not sure how much of an impact this has but may as well not include it for the other two projects.

Reply:
Yeah, that's a good idea. I'll move it there and merge.